psh/mem/mem.c

```diff
@@ -343,8 +343,9 @@ static int psh_mem(int argc, char **argv)
 		}
 	}
 
-	if (optind < argc) {
-		fprintf(stderr, "mem: unknown argument: %s\n", argv[optind]);
+	if (optind <= argc) {
+		ind = (optind == argc) ? optind - 1 : optind;
```

Comment:
Providing the only option causes early return `optind == argc` will be `--` case.